test/specs/Calculator.spec.js

```diff
@@ -0,0 +1,75 @@
+const Calculator = require("../../app/Calculator");
+const { expect } = require("chai");
+
+describe("scenarios for 'add' method", function () {
+  let validator;
+  beforeEach(function () {
+    validator = new Calculator();
```

Comment:
You are not change the internal state of the calculator instance using methods `add()` and `multiply()`. You can simply initialize calculator in the line 5.